internal/dashboard/static/style.css

```
@@ -1,170 +1,458 @@
+
+#feed-list::-webkit-scrollbar-thumb {
+  background: var(--border);
+  border-radius: 3px;
+}
+
+#feed-list::-webkit-scrollbar-thumb:hover {
+  background: var(--text-muted);
+}
+
+/* ── feed entries ── */
+@keyframes slideIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)